### PR TITLE
Fix(inventory): add history on Computer_Item add when Mainasset already exist

### PR DESCRIPTION
**Pull request by stonebuzz**
Add missing ```Log``` when ```Computer_Item``` is added (and when ```MainAsset``` exist because we log nothing on first inventory)

image

NB : when ```Computer_Item``` is removed ```Log``` is well added 

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | [433](https://github.com/glpi-project/glpi-inventory-plugin/issues/433)

### CODE REVIEW

**Comment by trasher** [comment]
tests are broken

File: src/Inventory/Asset/InventoryAsset.php [hunk 1]
```diff
@@ -408,7 +420,7 @@ protected function addOrMoveItem(array $input): void
         }
 
         $citem = new \Computer_Item();
-        $citem->add($input, [], false);
+        $citem->add($input, [], !$this->getItem()->isNewItem()); //log only if mainitem is not new
```
Comment (trasher) [inline src/Inventory/Asset/InventoryAsset.php:423]: Is the new method required? This should work:
```suggestion
        $citem->add($input, [], !$this->item->isNewItem()); //log only if mainitem is not new
```

Comment (stonebuzz) [reply src/Inventory/Asset/InventoryAsset.php]: indeed =)

File: install/migrations/update_10.0.10_to_10.0.11/socket.php [hunk 2]
```diff
@@ -0,0 +1,167 @@
+<?php
```
Comment (trasher) [inline install/migrations/update_10.0.10_to_10.0.11/socket.php:1]: Should not have been added :)

**Review by trasher — APPROVED** [review]

File: tests/functional/Glpi/Inventory/Assets/Monitor.php [hunk 3]
```diff
@@ -973,4 +973,165 @@ public function testInventoryImportOrNot()
...
+        ]);
+        $this->integer(count($logs))->isIdenticalTo(1);
+
+        //we have 2 monitor
+        $this->integer(count($monitor->find(['NOT' => ['name' => ['LIKE', '_test_%']]])))->isIdenticalTo(2);
+        //we have 2 monitor items linked to the computer
+        $this->integer(count($item_monitor->find(['itemtype' => 'Monitor', 'computers_id' => $computers_id])))->isIdenticalTo(2);
+        //we have 2 dynamic link
+        $this->integer(count($item_monitor->find(['itemtype' => 'Monitor', 'computers_id' => $computers_id, 'is_dynamic' => 1])))->isIdenticalTo(2);
+
```
Comment (trasher) [inline tests/functional/Glpi/Inventory/Assets/Monitor.php:1135]:
```suggestion
```

**Pull request merged**